packages/plugin-commands-publishing/src/publish.ts

```diff
@@ -68,7 +69,23 @@ export async function handler (
   } & Pick<Config, 'allProjects' | 'selectedProjectsGraph' | 'gitChecks'>,
 ) {
   if (opts.gitChecks && await isGitRepo()) {
-    await gitChecks()
+    if (await getCurrentBranch() !== 'master') {
```

Comment:
I think we need a way to allow a different branch name. Some use `latest` instead of `master`. So there should be a config setting like `publish-branch`.